### PR TITLE
Remove Wagtail 1.13 support, add Django 2.2 import support, and add Black for autoformatting

### PR DESCRIPTION
**Pull request by cwdavies**
## Additions
Support for Django 2.2 imports
Add black to tox

## Removals
Support for Wagtail 1.13 in tox and travis
`from __future__ import absolute_import, unicode_literals`

## Checklist

* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [x] Passes all existing automated tests

### CODE REVIEW

File: pip-wheel-metadata/wagtail_sharing.dist-info/LICENSE [hunk 1]
```diff
@@ -0,0 +1,121 @@
+Creative Commons Legal Code
```
Comment (willbarton) [inline pip-wheel-metadata/wagtail_sharing.dist-info/LICENSE]: I don't think y'all meant to commit this file?

File: pip-wheel-metadata/wagtail_sharing.dist-info/top_level.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+wagtailsharing
```
Comment (willbarton) [inline pip-wheel-metadata/wagtail_sharing.dist-info/top_level.txt]: Or this file?

File: pip-wheel-metadata/wagtail_sharing.dist-info/LICENSE (hunk 1 above)
Comment (chosak) [inline pip-wheel-metadata/wagtail_sharing.dist-info/LICENSE]: Why is this file being added here?

File: pip-wheel-metadata/wagtail_sharing.dist-info/METADATA [hunk 3]
```diff
@@ -0,0 +1,206 @@
+Metadata-Version: 2.1
```
Comment (chosak) [inline pip-wheel-metadata/wagtail_sharing.dist-info/METADATA]: Why is this file being added here?

File: pip-wheel-metadata/wagtail_sharing.dist-info/top_level.txt (hunk 2 above)
Comment (chosak) [inline pip-wheel-metadata/wagtail_sharing.dist-info/top_level.txt]: Why is this file being added here?

File: tox.ini [hunk 4]
```diff
@@ -22,11 +21,40 @@ deps=
...
+    flake8 wagtailsharing
+    isort --check-only --diff --recursive wagtailsharing
+
+[flake8]
+ignore=E731,W503,W504
+exclude=
+    .git,
+    .tox,
+    __pycache__,
+    node_modules,
```
Comment (chosak) [inline tox.ini]: This line is not needed here as there is no frontend build in this package.

File: wagtailsharing/tests/test_urls.py [hunk 5]
```diff
@@ -1,20 +1,20 @@
...
+
+import wagtail.core.urls as wagtail_core_urls
+import wagtailsharing.urls
+from mock import patch
+
 
 try:
     from importlib import reload
 except ImportError:
     pass
```
Comment (willbarton) [inline wagtailsharing/tests/test_urls.py]: `importlib.reload` is available from 3.4 on, so this `try`/`except` can be removed.

```suggestion
from importlib import reload
```

**Comment by Scotchester** [comment]
@willbarton @chosak Not sure where the pip-wheel-metadata additions came from, but I'll remove them.

**Comment by Scotchester** [comment]
And I swear we tested the linting locally before pushing 🤔

**Comment by Scotchester** [comment]
I'll rebase these edits from review back into a cleaner history.

**Comment by Scotchester** [comment]
> Would it be possible to split this up into two separate PRs, one for adding black and one for these version changes? The black changes are quite extensive which makes it hard to review the version changes. It'd be nice to review these separately, and separate PRs feels like a better place than just separate commits.

I appreciate the suggestion, and I think we'll do them separately in the future, but we discussed it earlier today in our team and agreed that the same PR was fine as long as the commits were separate and you could easily ignore the Black commit, and I went to a lot of trouble to get it neat and tidy that way, so I'd prefer not to split it out now :)

**Pull request merged**